library/swift/test/StatsClientImplTests.swift

```diff
@@ -19,7 +19,7 @@ final class StatsClientImplTests: XCTestCase {
     let mockEngine = MockEnvoyEngine()
     let statsClient = StatsClientImpl(engine: mockEngine)
     let counter = statsClient.counter(elements: ["test", "stat"])
-    counter.increment()
+    XCTAssertEqual(counter.increment(), 0)
```

Comment:
@rebello95 I was not able how to import kEnvoySuccess here

Reply:
If you `@testable import EnvoyEngine` it should work I believe

Reply:
worked, thanks!